lib/src/mixins/batched_redraws.dart

```diff
@@ -39,6 +41,9 @@ class _RedrawScheduler implements Function {
 
         (component as react.Component)?.setState({}, chainedCallbacks);
       })
+      ..forEach((component, callbacks) {
```

Comment:
I can't wrap my head around why you have to iterate again. Can you help me understand that?